scripts/linters/js_ts_linter.py

```diff
@@ -210,21 +209,20 @@ def _get_expressions_from_parsed_script(self):
 
         return parsed_expressions_in_files
 
-    def _compile_ts_file(self, filepath, dir_path):
-        """Compiles a typescript file and returns the path for compiled
-        js file.
-        """
-        cmd = (
-            './node_modules/typescript/bin/tsc -outDir %s -allowJS %s '
```

Comment:
@vojtechjelinek @Hudda are all the arg removals from this command ok?

(Also if so, why were they there in the first place?)

Reply:
They are, since these are now defined in `tsconfig.json`.